src/common/mailFunctionality/SharedMailUtils.ts

```diff
@@ -343,192 +206,68 @@ export function checkAttachmentSize(files: ReadonlyArray<Attachment>, maxAttachm
 	}
 }
 
-/**
- * @returns {boolean} true if the given mail was already replied to. Otherwise false.
- * Note that it also returns true if the mail was replied to AND forwarded.
- */
-export function isRepliedTo(mail: Mail): boolean {
-	return mail.replyType === ReplyType.REPLY || mail.replyType === ReplyType.REPLY_FORWARD
-}
-
 export enum RecipientField {
 	TO = "to",
 	CC = "cc",
 	BCC = "bcc",
 }
 
-export type FolderInfo = { level: number; folder: MailFolder }
-
-export async function getMoveTargetFolderSystems(model: MailModel, mails: readonly Mail[]): Promise<Array<FolderInfo>> {
-	const firstMail = first(mails)
-	if (firstMail == null) return []
-
-	const mailboxDetails = await model.getMailboxDetailsForMail(firstMail)
-	if (mailboxDetails == null) {
-		return []
-	}
-	const folderSystem = mailboxDetails.folders
+export function isTutanotaMailAddress(mailAddress: string): boolean {
```

Comment:
The list was so nicely renamed, could this function get the same treatment? :innocent: